docs/source/install.rst

```diff
@@ -123,3 +123,35 @@ channel:
 .. _`Arch Linux` : https://www.archlinux.org/
 .. _`Hitchhiker's Guide to Python` : http://docs.python-guide.org/en/latest/
 .. _`Homebrew` : http://brew.sh
+
+Installing in ``conda`` environments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is recommended to install Zipline in an isolated ``conda`` environment. 
+Installing Zipline in ``conda`` environment will not interfere your default 
+Python deployment or site-packages, which will prevent any possible conflict 
+with your global libraries. For more information on ``conda environment``, see 
+the `Conda User Guide <https://conda.io/docs/user-guide/tasks/manage-environments.html>`_
+
+Assuming ``conda`` has been set up, you can create a virtual environment:
+
+.. code-block:: bash
+
+    $ conda create -n zipline
```

Comment:
It's possible I'm remembering incorrectly, but something that might be confusing at this step is that users will end up with a conda env with a python version that matches their root env. If they installed latest conda, then I think they'll have python3.6, for which we don't currently have zipline packages. So probably we should specify the python version for the newly created env or explain this situation.